angular/projects/admin-nrpti/src/app/records/inspections/inspection-detail/inspection-detail.component.html

```diff
@@ -0,0 +1,100 @@
+<main class="detail-page">
+  <div>
+    <section class="content-section">
+      <div class="content-grid">
+        <div class="grid-section-header">
+          <div class="grid-section-header-title">
+            <span class="grid-section-header-title-text">Shared Data [Master]</span>
+            <span class="grid-section-header-title-sub-text"
+              >Last Edited {{ (data && data._master.dateUpdated | date: 'mediumDate') || '-' }}</span
```

Comment:
Can we close > on within the same tag. Same for all occurrences below.

Reply:
Done.

I also updated the formatter to default to that style.